metals/src/main/scala/scala/meta/internal/metals/Embedded.scala

```diff
@@ -61,44 +60,59 @@ final class Embedded(
 }
 
 object Embedded {
-  def downloadSettings(
+  def fetchSettings(
       dependency: Dependency,
       scalaVersion: String
-  ): Settings =
-    new coursiersmall.Settings()
-      .withTtl(Some(Duration.Inf))
-      .withDependencies(List(dependency))
+  ): Fetch = {
+    val fetch = Fetch
+      .create()
+      .withDependencies(List(dependency): _*)
```

Comment:
just passing the `withDependency(dependency)` doesn't work?

Reply:
no, it expects varargs